src/osu_2019/pp.rs

```diff
@@ -247,8 +247,8 @@ impl<'m> OsuPP<'m> {
         let effective_miss_count = self.calculate_effective_miss_count();
 
         // NF penalty
-        if self.mods.nf() {
-            multiplier *= 0.9_f32.max(1.0 - 0.2 * effective_miss_count);
+        if self.mods.nf() && !self.mods.rx() {
```

Comment:
this line is useless since this calc is used only for relax, you can just remove this entire block actually